catalog/log-export/folder/bigquery-export/custom/Kptfile

```diff
@@ -0,0 +1,21 @@
+apiVersion: kpt.dev/v1
```

Comment:
I'm not sure about splitting this into subpackages. I definitely don't want users to have to update the Kptfile for each subpackage separately.

Reply:
"update the Kptfile for each subpacakge" --> That will be less a concern with [variant constructor](https://github.com/GoogleContainerTools/kpt/issues/2590)

Reply:
Variant constructors are more of a bandaid. They don't solve for the intrinsic complexity of managing deeply nested packages.